src/components/ProgramRecordsList.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+
+import { ChevronLeft, Info } from '@edx/paragon/icons';
+import { Alert, Hyperlink } from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { getAuthenticatedUser } from '@edx/frontend-platform/auth';
+import { getConfig } from '@edx/frontend-platform/config';
+import { logError } from '@edx/frontend-platform/logging';
+import _ from 'lodash';
+import getProgramRecords from './data/service';
```

Comment:
nit: There is convention for putting an empty line between things you’re importing in from external libraries and things you’re importing from elsewhere in this project. If you want to follow that, you could put a space between lines 9 and 10—but really it doesn’t matter. :)